types/mongoose-aggregate-paginate-v2/index.d.ts

```diff
@@ -0,0 +1,90 @@
+// Type definitions for mongoose-aggregate-paginate-v2 1.0
+// Project: https://github.com/webgangster/mongoose-aggregate-paginate-v2
+// Definitions by: Linus Brolin <https://github.com/linusbrolin>
```

Comment:
I assume all these people know they're being signed up as owners?

Reply:
Actually these people are the original contributors/owners of the mongoose-paginate-v2. Also, @knyuwork and @tuliren  where the authors of the file I used as an inspiration on an issue in the project.

So, I can remove them if it is a better idea, but I wanted to give them attribution for their work.

Reply:
I really appreciate your attempt to ensure attribution, but this is actually a list of people interested in reviewing subsequent changes to these types.

Reply:
Ok, I've removed the owners and added an acknowledgement instead.